<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="3520dell" failures="1" tests="4" name="Test Runner 1" time="0.09" errors="0" timestamp="2024-11-08T13:12:04 IST">
  <testcase classname="distributedparallel.test4" name="demo" time="0.018"/>
  <testcase classname="distributedparallel.test4" name="demo2" time="0.004">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at distributedparallel.test4.demo2(test4.java:16)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1570)
... Removed 18 stack frames]]>
    </failure>
  </testcase> <!-- demo2 -->
  <testcase classname="distributedparallel.test3" name="demo" time="0.006"/>
  <testcase classname="distributedparallel.test3" name="demo2" time="0.003"/>
</testsuite> <!-- Test Runner 1 -->
